app/controllers/photos_controller.rb

```diff
@@ -134,6 +119,22 @@ def destroy
 
   private
 
+  def valid_models
+    ["planting", "harvest", "garden"]
+  end
+
+  def has_valid_key
```

Comment:
The other thing to ponder is concerns. I'm not sure _exactly_ how they'd
change things as I haven't tried them yet.

On Wed, May 25, 2016 at 10:28 AM pozorvlak <notifications@github.com> wrote:

> In app/controllers/photos_controller.rb
> <https://github.com/Growstuff/growstuff/pull/905#discussion_r64582547>:
>
> > @@ -134,6 +119,22 @@ def destroy
> >
> >    private
> >
> > +  def valid_models
> > +    ["planting", "harvest", "garden"]
> > +  end
> > +
> > +  def has_valid_key
>
> Oh, right, you've moved the checks into these helper methods. Sorry!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Growstuff/growstuff/pull/905/files/b601fe40b3952649f0ffe4b9fd9f31ea09a5a6cc#r64582547>
>